proto/executor.proto

```diff
@@ -105,6 +105,7 @@ message Aggregation {
 	// Aggregate functions.
 	repeated Expr agg_func = 2;
 	repeated RpnExpr rpn_agg_func = 5;
+	optional ExprType func_type = 7;
```

Comment:
I suspect a single `ExprType` might not be enough , for example, the field type is missing